lms/static/scripts/frontend_apps/config.ts

```diff
@@ -93,8 +93,10 @@ export type FilePickerConfig = {
   };
   canvas: {
     enabled: boolean;
+    pagesEnabled: boolean;
     foldersEnabled: boolean;
     listFiles: APICallInfo;
+    listPages: APICallInfo;
```

Comment:
Is this right? listPages might not be present at all if pagesEnabled == False.

Reply:
See https://github.com/hypothesis/lms/pull/5716/files#r1342436510